projects/packages/my-jetpack/_inc/data/purchases/use-purchases.ts

```diff
@@ -0,0 +1,34 @@
+import { useQuery } from '@tanstack/react-query';
+import apiFetch from '@wordpress/api-fetch';
+import { __ } from '@wordpress/i18n';
+import { REST_API_SITE_PURCHASES_ENDPOINT, REST_API_SITE_PURCHASES_QUERY_KEY } from '../constants';
+import useNotice from '../notices/use-notice';
+import type { Purchase } from '../../../global';
+import type { UseQueryResult } from '@tanstack/react-query';
+
+const cacheKey = REST_API_SITE_PURCHASES_QUERY_KEY;
+
+const usePurchases: () => UseQueryResult< Array< Purchase >, Error > = () => {
+	const queryResult = useQuery( {
+		queryKey: [ cacheKey ],
+		queryFn: async () =>
+			( await apiFetch( { path: REST_API_SITE_PURCHASES_ENDPOINT } ) ) as Array< Purchase >,
```

Comment:
We don't need `async ... await` here I think
```suggestion
		queryFn: () => apiFetch( { path: REST_API_SITE_PURCHASES_ENDPOINT } ) as Array< Purchase >,
```

Reply:
You are right, I must have added this when the query wasn't working and I was still trying things